src/az_iot_adu_client.c

```diff
@@ -0,0 +1,843 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
```

Comment:
Are these files submoduled from the emb c repo?

Reply:
No these have to be directly copied in. We basically clone locally, flatten the include paths and the src directories so they all live in one place. It's an Arduino requirement we have to abide by.

Reply:
makes sense - we'll have to add in the related files changes once those go in.
For review purposes, these files should match what's existing in the other repo though?

Reply:
correct everything under the `src/` directory is dumped in via a script. The only manipulation that is done is the script changes deep include paths to shallow ones. So `#include azure/iot/az_iot.h` becomes `#include az_iot.h`